docs/spec/typeddict.rst

```diff
@@ -737,26 +729,39 @@ Subclasses can combine these rules::
     class User(OptionalIdent):
         ident: str  # Required, mutable, and not an int
 
-Note that these are just consequences of structural typing, but they are highlighted here as the behavior now differs from the rules specified in :pep:`589`.
+Note that these are just consequences of :term:`structural` typing, but they
+are highlighted here as the behavior now differs from the rules specified in
+:pep:`589`.
 
-Type consistency
-^^^^^^^^^^^^^^^^
+Assignability
+^^^^^^^^^^^^^
 
-*This section updates the type consistency rules described above that were created prior to the introduction of ReadOnly*
+*This section updates the assignability rules described above that were created
+prior to the introduction of ReadOnly*
 
-A TypedDict type ``A`` is consistent with TypedDict ``B`` if ``A`` is structurally compatible with ``B``. This is true if and only if all of the following are satisfied:
+A TypedDict type ``B`` is :term:`assignable` to a TypedDict type ``A`` if ``B``
+is :term:`structurally <structural>` assignable to ``A``. This is true if and
+only if all of the following are satisfied:
```

Comment:
you could avoid saying "they're assignable because they're assignable" here ;)

```suggestion
is :term:`structurally <structural>` :term:`consistent` with ``A``. This is
true if and only if all of the following are satisfied:
```

Reply:
Maybe so, but the suggested way of saying it would be incorrect, since "structurally assignable" and "structurally consistent with" don't mean the same thing.

I could say "consistent subtype of", but that's just choosing to use our less-preferred synonym for assignable; I'm not sure that's an improvement.

I think in this case I want to say what it currently says: "typeddicts are assignable if they are structurally assignable, and here's what that means:"

The current way of wording that reads as a bit repetitive, but I think it's clear, which is the most important thing.